doc/getting-started/apl-nuc.rst

```diff
@@ -398,3 +401,53 @@ Set up Reference UOS
       :align: center
       :name: gsg-successful-boot
 
+
+.. _quick-setup-guide:
+
+Use the script to set up ACRN automatically
+===========================================
+
+It is little complicate to setup the SOS or UOS, so we provide a script to do it quickly and automatically.
+You can find the script `here
+<https://raw.githubusercontent.com/projectacrn/acrn-hypervisor/master/doc/getting-started/gsg_quick_setup.sh>`__
+and please note that should be run with root privilege since it will modify various system parameters.
+
+#. Installing Clear Linux and login system
+
+#. Open a terminal
+
+#. Run ``cd ~ && wget https://raw.githubusercontent.com/projectacrn/acrn-hypervisor/master/doc/getting-started/gsg_quick_setup.sh``
+
+#. Run ``sudo sh gsg_quick_setup.sh -s 28920`` to setup SOS
```

Comment:
I get a "You're trying to install an older version of Clear Linux" message with this command line. It looks like if I'm running a current version of Clear Linux and I want to get ACRN installed, this script won't do that.  Will the script only install and setup ACRN if I upgrade the Clear Linux version too?

Reply:
Yes, so far it's not support downgrade SOS, but we still able to run this script to set up ACRN for current version of Clear Linux.